model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -27,6 +27,8 @@ import com.here.ort.model.config.PathExclude
 import java.util.SortedSet
 
 internal class LicenseResolver(private val ortResult: OrtResult) {
+    private val curationMatcher = FindingCurationMatcher()
```

Comment:
Nit: Would have been nice to read in the commit message "So that not a new instance gets created for every single map operation."

Reply:
My main reason for this change was actually readability.

Reply:
Oh. In my opinion it even slightly *decreases* readability as the code becomes more complex (= spread across multiple lines) now. But the outcome is good anyway and worth the cost, as instances should not be created in loops.

Would you agree that the purpose of not creating instances in loops is the "higher" / more important goal than (arguable) readability, and thus be fine with squashing the commits?

Reply:
I think this topic is not worth discussing, I would prefer to just keep the commits as is to not spent any more time on it. If you insist on squashing it I will do it.

Reply:
This time, I insist on it indeed. With a proper commit message about not creating instances in inner loops, please.

Reply:
Done.